install-dependencies.sh

```diff
@@ -203,6 +203,22 @@ node_exporter_url() {
     echo "https://github.com/prometheus/node_exporter/releases/download/v$NODE_EXPORTER_VERSION/$(node_exporter_filename)"
 }
 
+declare -A MINIO_ARCH=(
+    ["x86_64"]=amd64
+    ["aarch64"]=arm64
+    ["s390x"]=s390x
```

Comment:
can we consolidate this with `NODE_EXPORTER_ARCH` to something like
```bash
declare -A GO_ARCH=(
     ["x86_64"]=amd64
     ["aarch64"]=arm64
     ["s390x"]=s390x
)
```
see https://github.com/golang/go/blob/b94dc384cabf75e7e8703265cd80f5324f84b642/src/go/build/syslist.go#L55

as both minio and node_exporter are implemented in golang, and what they are using is actually `GOARCH`, and `GOOS`, which is always "linux" here.

Reply:
It's a good idea, but can be done in a follow up.

Reply:
proposed at #11359 13151